debian/radxa-system-config-allwinner.hdmi-toggle-once.service

```
@@ -0,0 +1,14 @@
+[Unit]
+Description=HDMI toggle (1280x1024 -> 1920x1080) once after boot
+After=display-manager.service sddm.service
```

```suggestion
After=display-manager.service
```
sddm/gdm会自动注册成display-manager